modules/openapi-generator/src/main/resources/Java/libraries/jersey2/ApiClient.mustache

```diff
@@ -93,6 +94,10 @@ public class ApiClient {
     authentications.put("{{name}}", new OAuth());{{/isOAuth}}{{/authMethods}}
     // Prevent the authentications from being modified.
     authentications = Collections.unmodifiableMap(authentications);
+
+    // Setup authentication lookup (key: authentication alias, value: authentication name)
+    authenticationLookup = new HashMap<String, String>();{{#authMethods}}{{#vendorExtensions.x-lookup}}
+    authenticationLookup.put("{{name}}", "{{.}}");{{/vendorExtensions.x-lookup}}{{/authMethods}}
```

Comment:
I agree with comments made in #4940 that `x-lookup` isn't intuitive. I like William's suggest in that PR of `x-auth-id-alias`.

Reply:
Thanks. I have renamed the extension.